internal/cmd/stack/run_prioritize.go

```diff
@@ -0,0 +1,52 @@
+package stack
+
+import (
+	"fmt"
+	"github.com/shurcooL/graphql"
+	"github.com/spacelift-io/spacectl/internal/cmd/authenticated"
+	"github.com/urfave/cli/v2"
+)
+
+type setRunPriorityMutation struct {
+	SetRunPriority struct {
+		ID string `graphql:"id"`
+	} `graphql:"runPrioritizeSet(stack: $stackId, run: $runId, prioritize: $prioritize)"`
+}
+
+func runPrioritize(cliCtx *cli.Context) error {
+	stackID, err := getStackID(cliCtx)
+	if err != nil {
+		return err
+	}
+	runID := cliCtx.String(flagRequiredRun.Name)
+
+	var mutation setRunPriorityMutation
+
+	variables := map[string]interface{}{
+		"stackId":    graphql.ID(stackID),
+		"runId":      graphql.ID(runID),
+		"prioritize": graphql.Boolean(true),
```

Comment:
Should we extend this so that the caller could set this? This customer don't have to go to UI to revert their action in case they make a mistake.

Reply:
In my opinion, from the user's perspective, it would be easier to have a separate command "deprioritize". What do you think?

Reply:
f6fcd7349259762110b8a9782db8683fda2e3b8b